clients/src/test/java/org/apache/kafka/clients/consumer/internals/MembershipManagerImplTest.java

```diff
@@ -29,6 +29,8 @@
 import org.apache.kafka.common.TopicPartition;
```

Comment:
I wonder if we could add the example that we discussed offline as a test:
```
0: [T1, T2] -- T2 unresolved (only T1 is reconciled)
1: [T1, T2, T3] -- T2 unresolved (skipped, since reconciliation in progress)
2: [T1, T2] -- T2 unresolved
```
What do you think?

Reply:
Agree it's not covered. (Happy to take that separately right after this if you prefer @lucasbru...knowing you're getting busy very soon)